.github/workflows/release-binary.yml

```diff
@@ -42,7 +42,11 @@ jobs:
         with:
           version: latest
           args: release -f .goreleaser/linux-amd64.yml --clean
-          workdir: /
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          SLACK_WEBHOOK: "${{ secrets.RELEASE_SLACK_WEBHOOK }}"
+          DISCORD_WEBHOOK_ID: "${{ secrets.DISCORD_WEBHOOK_ID }}"
+          DISCORD_WEBHOOK_TOKEN: "${{ secrets.DISCORD_WEBHOOK_TOKEN }}"
```

Comment:
GitHub Access Token is required to create a release.
Also, in linux-amd64.yml slack and discord integrations are enabled.

https://github.com/projectdiscovery/naabu/blob/a687d975c0bb7791bef01ccdfc2cdd6962bc062b/.goreleaser/linux-amd64.yml#L28-L37